### DISCUSSION TITLE
How to disable this popup?

### DISCUSSION THREAD

**Discussion by Edfgdz** [bromite/bromite#1156] [Q&A]
I have installed twitter app or google apps on my phone so when i open twitter website or google website such as google play store then it will popup this windows requesting me open those pages in apps. 

So is there any way to let these popups not showing then i could go into those websites directly?

![Screenshot_2021-05-22-07-19-18-406_android](https://user-images.githubusercontent.com/26460124/119207243-21534000-bad0-11eb-8790-8c9e7ee7f8bd.jpg)

![Screenshot_2021-05-22-07-19-38-155_android](https://user-images.githubusercontent.com/26460124/119207246-23b59a00-bad0-11eb-88b9-f8d8d4b11acf.jpg)

**Comment by ghost** [comment]
remove all unnecessary applications, and better the firmware itself.

**Reply by Edfgdz** [reply]
Wish there are other solutions without uninstalling apps

**Comment by Edfgdz** [comment]
@csagan5 

Hello developer,is it possible to let this browser not popup this windows without uninstalling apps?

**Reply by uazo** [reply]
seems it's possibile only with a change in

https://source.chromium.org/chromium/chromium/src/+/main:components/external_intents/android/java/src/org/chromium/components/external_intents/ExternalNavigationHandler.java;l=1168;drc=6b9c0c306067c1e1524d204961587e96eafd3ef7;bpv=1;bpt=1

**Reply by Edfgdz** [reply]
So is it possible to let that window disappear based on this in Bromite future updates?

**Reply by csagan5** [reply]
@uazo the switch is `--disable-external-intent-requests` but there is no flag for it; I will add a flag.
